0.data_analysis/README.md

```diff
@@ -0,0 +1,6 @@
+# 0.data_analysis
+This module focuses on exploring relationships in cell morphology data not included in the [nf1_cellpainting_data](https://github.com/WayScience/nf1_cellpainting_data) repository.
+We organize this module into two correlation analyses:
+
+1. `plate_5_characteristics` explores similarities between genotype groups using cell morphologies aggregated to the well-level, post-feature selection.
+2. `all_plates_characteristics` explores similarities between the wells of plates 3, 3 prime, 4, and 5 using genotype and seeding density groups.
```

Comment:
Recommend updating this to remove plate 4 since we don't use it for this manuscript (beyond scope).

Reply:
I think think plate 4 is used in that analysis. Do you think we need to update this analysis? If so, should we update it in the pr, or a later pr?

Reply:
Read your previous comment later on (below), so I am resolving this conversation